### PR TITLE
Feature/46 use django simple certmanager library

### LINKED ISSUE #46: Removing internal cert-manager and use django-simple-certmanager library instead
See step 2 of https://github.com/open-formulieren/open-forms/issues/1849 

Library used: https://github.com/maykinmedia/django-simple-certmanager 

- [x] Remove related code to cert-manager
- [x] Update documentation
- [x] Add data migrations

### PR DESCRIPTION
**Pull request by Ewen-Lgy**
Fixes #46 

Extracting the certficate manager to a standalone library

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,9 +1,10 @@
 Changes
 =======
 
-0.20.0 (2022-08-23)
+0.20.0 (2022-8-22)
 -------------------
 
+* Moving TLS certificate management to another library called``django-simple-certmanager``. Managing Certificates with this library remains possible for now, but please consider to update your project.
```
Comment (sergei-maertens) [inline CHANGELOG.rst:7]: you'll need a separate changelog entry for this - 0.20.0 has been released to PyPI on August 23rd already.

File: docs/index.rst [hunk 2]
```diff
@@ -20,7 +19,7 @@ Developed for use in Dutch government software where data exchange with external
 services is run of the mill, this library provides flexibility in configuring your
 environment(s).
 
-ZGW Consumers allows you to centralize the location, credentials, TLS certificate... to
+ZGW Consumers allows you to centralize the location, credentials... to
```
Comment (sergei-maertens) [inline docs/index.rst:22]:
```suggestion
ZGW Consumers allows you to centralize the location, credentials, API schema information... to
```

File: docs/quickstart.rst [hunk 3]
```diff
@@ -18,8 +18,7 @@ Installation
 2. Add ``zgw_consumers`` to the ``INSTALLED_APPS`` setting.
 3. Run ``python src/manage.py migrate`` to create the necessary database tables
 4. Configure `django-privates <https://django-privates.readthedocs.io/en/latest/quickstart.html>`_
```
Comment (sergei-maertens) [inline docs/quickstart.rst:20]: I'd point to django-simple-certmanager instead - zgw-consumers itself doesn't need django-privates as dependency anymore I think.

File: setup.cfg [hunk 4]
```diff
@@ -41,6 +41,7 @@ install_requires =
     django-relativedelta
     django-solo
     gemma-zds-client>=0.14.0
+    django-simple-certmanager
```
Comment (sergei-maertens) [inline setup.cfg:44]: django-privates can be removed from this dependency list

File: zgw_consumers/migrations/0016_auto_20220818_1412.py [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+                blank=True,
+                help_text="The SSL/TLS certificate of the server",
+                null=True,
+                on_delete=django.db.models.deletion.PROTECT,
+                related_name="service_server",
+                to="simple_certmanager.certificate",
+            ),
+        ),
+        migrations.RunPython(migrations.RunPython.noop, get_operation(forwards=False)),
+    ]
```
Comment (sergei-maertens) [inline zgw_consumers/migrations/0016_auto_20220818_1412.py:63]: we'll have to see if we can combine data and schema migrations in the same transaction, I remember having issues with that on postgres sometimes.

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
Can you also update the translation files, there should be some that are gone now.

``django-admin makemessages -l nl`

File: docs/quickstart.rst (hunk 3 above)
Comment (Ewen-Lgy) [reply docs/quickstart.rst:20]: I agree

File: zgw_consumers/migrations/0016_auto_20220818_1412.py (hunk 5 above)
Comment (Ewen-Lgy) [reply zgw_consumers/migrations/0016_auto_20220818_1412.py:63]: I merged the migrations and it seems to be working fine

**Review by sergei-maertens — APPROVED** [review]
Tested the forwards and reverse (data) migration on Open Forms project and it works as expected :+1:

**Pull request merged**